src/routes/safe/components/Transactions/helpers/TxParametersDetail/index.tsx

```diff
@@ -81,14 +77,14 @@ export const TxParametersDetail = ({
     }
   }, [lastQueuedTxNonce, nonce, safeNonceNumber])
 
-  if (!isTransactionExecution && !isTransactionCreation && isOffChainSignature) {
-    return null
-  }
-
   const onChangeExpand = () => {
     setIsAccordionExpanded(!isAccordionExpanded)
   }
 
+  if (parametersStatus === 'DISABLED') {
```

Comment:
This replaces `if (!isTransactionExecution && !isTransactionCreation && isOffChainSignature) `. This logic is already done outside of this component and is passed down as `parametersStatus`.